job_spec/config/compute_environments.yml

```diff
@@ -1,9 +1,33 @@
+# Compute Environments that are shared between multiple jobs and/or steps should be defined here, 
+# rather than in the job spec. The format is the same, except the name becomes the key:
+
+# ExampleComputeEnvName:
+#   instance_types: ...
+#   ami_id: ...
+#   allocation_type: ...
+#   allocation_strategy: ...
+
+# Once the environment is defined here, it can be imported in the job spec step
+# by using the `import` key:
+
+# JOB_NAME:
+#   ...
+#   steps:
+#     step_name:
+#       ...
+#       compute_environment:
+#         import: 'ExampleComputeEnvName'
+
+# Note that the name must be unique, i.e. a compute environment that is defined in a job spec using
+# the `name` key cannot also be defined here, unless it is not imported in the same deployment (envs here
+# are only rendered if they are imported). Also note that the name should be in pascal case to avoid
+# cfn-lint errors.
```

Comment:
This paragraph was somewhat difficult for me to parse -- is it just describing error cases that will be caught automatically? i.e. duplicate names should be caught by `render_cf.py`, and I assume non-pascal-case names would also lead to an error? If so, I suggest we just remove it and let those error messages speak for themselves.
```suggestion
```

Reply:
I mostly included that to highlight the 'unless it is not imported in the same deployment' part. I'll remove that if I remove the behavior.